R/calcOutput.R

```diff
@@ -13,7 +13,10 @@
 #' outputfolder as specified in the config.
 #' @param years A vector of years that should be returned. If set to NULL all
 #' available years are returned.
-#' @param round A rounding factor. If set to NULL no rounding will occur.
+#' @param round Number of decimal places to round to.  Ignored if `NULL`.  See
+#'   [`round()`] for details.
+#' @param signif Number of significant digits to round to.  Ignored if `NULL` or
+#'   if `round` is not `NULL`.  See [`signif()`] for details.
```

Comment:
I would not ignore it if round is being used.

Reply:
but if both can be used at the same time it would be good to indicate here in which order they are being applied

Reply:
Seemed like the option to me best preserving the original behaviour.
I don't see any use in using both `round()` and `signif()`, but the result would be the same no matter the order in which they are used.

Reply:
oh true, so ignore my comment about the order. I agree that it is hard to imagine a case where the application of both at the same time would make sense, but I would not preclude the possibility to do so as there are no technical reasons for it.